.changelogs/issue_2237.yml

```
@@ -0,0 +1,6 @@
+significance: patch
```

```suggestion
significance: minor
```
This can be a minor change rather than a `patch`